portal/templates/contract/components/bond-transactions-table.njk

```diff
@@ -73,10 +73,17 @@
 
                   {% set requestedCoverDateCanEdit = userIsMaker and bond.status !== "Maker's input required" and dealStatusAllowsRequestedCoverDateChange and dealSubmissionType === 'Manual Inclusion Application' and bond.facilityStage === 'Issued' and bond.issueFacilityDetailsSubmitted %}
 
-                  {% if requestedCoverDateCanEdit %}
-                    {{ requestedCoverStartDateLink.render(componentData) }}
-
-                    {% else %}
+                  {% if requestedCoverDateCanEdit or bond.canAmendFacility %}
```

Comment:
I feel like there should be a better way of doing this, I've ended up with a lot of ifs here which is a bit hard to follow.

I'm also not sure about the `<span>|</span>`. Is there a better way of adding the separator between links?

Reply:
I think this logic might be wrong, I was thinking that the `issueOrDeleteFacilityLink` couldn't appear at the same time as the amend link, but actually I think it might be able to, so it shouldn't be an `else`